runtime/src/iree/hal/drivers/vulkan/pipeline_layout.h

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef IREE_HAL_DRIVERS_VULKAN_NATIVE_PIPELINE_LAYOUT_H_
+#define IREE_HAL_DRIVERS_VULKAN_NATIVE_PIPELINE_LAYOUT_H_
```

Comment:
nit: update header guard to match new file name (here and at the bottom of the file)

```suggestion
#ifndef IREE_HAL_DRIVERS_VULKAN_PIPELINE_LAYOUT_H_
#define IREE_HAL_DRIVERS_VULKAN_PIPELINE_LAYOUT_H_
```